homeassistant/components/spaceapi/__init__.py

```diff
@@ -105,6 +107,27 @@ class APISpaceApiView(HomeAssistantView):
     url = URL_API_SPACEAPI
     name = 'api:spaceapi'
 
+    @staticmethod
+    def get_sensor_data(hass, spaceapi, sensor):
+        """get necessary data from senor"""
+        sensor_state = hass.states.get(sensor)
+        if not sensor_state:
+            return None
+        sensor_data = {
+            ATTR_NAME: sensor_state.name,
+            ATTR_VALUE: sensor_state.state
+        }
+        if ATTR_SENSOR_LOCATION in sensor_state.attributes:
```

Comment:
Which sensors contains the location in their attributes?

Reply:
As far as I know none does by default, but for our hackerspace (that uses the spaceapi component already) I've set the location attribute for the sensors to make clear where these sensors are located in the space. 
You can see it in action here: https://spaceapi.reaktor23.org/
And the [SpaceAPI Documentation](https://spaceapi.io/pages/docs.html) has the location marked as required, so I thought thats a nice way to provide the location.

Reply:
I see that for some sensors the `location` is required.